OmiseSwift/APIClient.swift

```diff
@@ -36,12 +36,22 @@ public class APIClient: NSObject {
     func preferredKeyForServerEndpoint(_ endpoint: ServerEndpoint) -> String {
         switch endpoint {
         case .api:
-            return config.secretKey
+            return config.secretKey + ":X"
         case .vault:
             return config.publicKey
         }
     }
     
+    func preferredEncodedKeyForServerEndpoint(_ endpoint: ServerEndpoint) throws -> String {
+        let data = preferredKeyForServerEndpoint(endpoint).data(using: .utf8, allowLossyConversion: false)
```

Comment:
Yeah you got a point. I just updated this PR to append `:X` on both types of keys and also add a comment on why we do that too.